README.md

```diff
@@ -6,10 +6,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `npm start`
+### `npm run start`
 
 Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Open [http://localhost:1234](http://localhost:1234) to view it in your browser.
```

Comment:
why 1234?

Reply:
create-react-app uses webpack with default port 3000.
parcel uses default port of 1234.
I have updated documentation accordingly.